language/move-prover/spec-lang/src/translate.rs

```diff
@@ -353,6 +353,56 @@ impl<'env> Translator<'env> {
             let pred_t = &Type::Fun(vec![param_t.clone()], Box::new(bool_t.clone()));
             let pred_num_t = &Type::Fun(vec![num_t.clone()], Box::new(bool_t.clone()));
 
+            // Transaction metadata
+            add_builtin(
+                self,
+                self.builtin_fun_symbol("sender"),
+                SpecFunEntry {
+                    loc: loc.clone(),
+                    oper: Operation::Sender,
+                    type_params: vec![],
+                    arg_types: vec![],
+                    result_type: address_t.clone(),
+                },
+            );
+
+            // constants (MAX_U8(), etc.)
+            add_builtin(
+                self,
+                self.builtin_fun_symbol("MAX_U8"),
```

Comment:
Since those are functions and not constants, I think we should use `max_u8` and so on for consistency.